indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -106,3 +107,30 @@ def _process_PUT(self):
     def _process_DELETE(self):
         session.user.favorite_rooms.discard(self.room)
         return '', 204
+
+
+# TODO
+class RHRoom(RHRoomBase):
+    def _process_args(self):
+        self.room = Room.get_one(request.view_args['room_id'])
+
+    def _process(self):
+        detailed_room = _make_detailed_room(self.room)
+        return jsonify(detailed_room)
+
+
+
+# TODO
+class RHRoomUpdate(RHRoomBase):
+    def _process_args(self):
+        self.room = Room.get_one(request.view_args['room_id'])
```

Comment:
RHRoomBase already does this (except for the `is_active` check, but that's only relevant later when doing the actual admin area since currently you only get the edit dialog for active rooms anyway since there's no admin room list yet where inactive rooms are shown)